go.sum

```diff
@@ -110,6 +110,8 @@ github.com/hexdigest/gowrap v1.1.7/go.mod h1:Z+nBFUDLa01iaNM+/jzoOA1JJ7sm51rnYFa
 github.com/hexdigest/gowrap v1.1.8/go.mod h1:H/JiFmQMp//tedlV8qt2xBdGzmne6bpbaSuiHmygnMw=
 github.com/hugelgupf/p9 v0.2.1-0.20230102222142-5cb68b4d03bb h1:rUzGqvx/TVHTtosnqlS8b4n7B4ClZ8emexHRethsWTQ=
 github.com/hugelgupf/p9 v0.2.1-0.20230102222142-5cb68b4d03bb/go.mod h1:1NBCX6Wq2cdaeDeagZuD5CygSuB5W5R5mnh/9e2DNPE=
+github.com/hugelgupf/p9 v0.2.1-0.20230803231326-7b9704081cf5 h1:uN6Q2LFhsP6mCEgmF12kAoWsuCHkjTO5zZ0gx2AKnd0=
```

Comment:
i'm a little ignorant of go.mod stuff.  are these 2 new lines supposed to replace the previous p9 mod lines?

Reply:
I am not clear either. They are generated by `go get github.com/hugelgupf/p9@main`.

Reply:
I wonder if go mod tidy would do a cleanup. The wonder and mystery of go.mod remain.

Reply:
`go mod tidy` works